crates/swc_stylis/tests/fixture/animation/output.css

```diff
@@ -40,8 +40,7 @@ animation-direction: alternate-reverse;
 -moz-animation: rotation-reverse;
 -o-animation: rotation-reverse;
 animation: rotation-reverse;
--webkit-animation: rotation-reverse reverse;
--moz-animation: rotation-reverse reverse;
--o-animation: rotation-reverse reverse;
```

Comment:
We should not prefix them, because reverse is part of new spec and will not work in prefixes